patches/codehook.slpatch

```diff
@@ -1,8 +1,3 @@
 [version=310, target=main]
-gsys::SystemTask::invokeDrawTV_+284 NOP // enable display debug stuff (which is used for hook)
-gsys::SystemTask::invokeDrawTV_+390:
-	MOV X1, X0
-	MOV X0, X25
-	BL render
-	B #0x2B8
-	
\ No newline at end of file
+StageScene::control+18:
+    BL teleportMarioIfNeeded
```

Comment:
This is the name of a function that we will define in our C++ code. We have not done so yet.